plugins/source/gcp/codegen/main.go

```diff
@@ -179,10 +179,15 @@ func generateResource(r recipes.Resource, mock bool) {
 		log.Fatal(fmt.Errorf("failed to create table for %s: %w", r.StructName, err))
 	}
 	if r.Multiplex == nil {
-		if _, ok := client.GcpServices[r.ServiceDNS]; !ok {
-			panic("unknown service DNS: " + r.ServiceDNS)
+		if r.ServiceDNS == "" {
```

Comment:
Doesn't this change means that if someone forgets to set `ServiceDNS` they'll use the wrong multiplexer? Instead the should set `Multiplex` directly if they need `ProjectMultiplex`.
If we want to make `ServiceDNS` optional we can use https://github.com/cloudquery/cloudquery/pull/5956/files

Reply:
If `ServiceDNS` is empty, then `ProjectMultiplex` is the only `Multiplex` option that will work

Reply:
Cool but should we always set it? If someone forgets to set it that specific service won't be enforce via the enabled services feature.
If someone wants to opt out from the enabled services feature they should set the Multiplex explicitly no?

Reply:
I mean if you want to use the `ProjectMultiplex` you can set that one directly instead of setting an empty `ServiceDNS`